haiku/src/implementation.cc

```diff
@@ -57,6 +57,8 @@ size_t ::iana_time_zone_haiku::get_tz(rust::Slice<uint8_t> buf) {
 
 #else
 
-size_t ::iana_time_zone_haiku::get_tz(rust::Slice<uint8_t>) { return 0; }
```

Comment:
If I'm reading this set of docs correctly, this declaration syntax is only available since C++17: https://en.cppreference.com/w/cpp/language/namespace.

Reply:
You're right! Could you do the same in line 13?

Reply:
done